brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-1254 Fix: Post Masonary - Taxonomy badge default background color not applying at the frontend

### Description
Taxonomy badge default background color not applying at the frontend

### Screenshots
Issue  -  https://d.pr/i/6zMpNz
Fix - https://share.getcloudapp.com/bLuBx4xe

### Types of changes

Bug fix (non-breaking change which fixes an issue)



### How has this been tested?
- Add a Post Masonry block
- Go to the general content setting
- Enable show taxonomy → Select taxonomy position above title → taxonomy style highlighted 
- Check backend and frontend color

### Checklist:
